components/ledger/pkg/storage/accounts.go

```diff
@@ -0,0 +1,63 @@
+package storage
+
+import (
+	"github.com/formancehq/stack/libs/go-libs/metadata"
+)
+
+type AccountsQuery OffsetPaginatedQuery[AccountsQueryPayload]
+
+type AccountsQueryPayload struct {
+	AfterAddress    string
```

Comment:
Should we add json tags on this struct since it will be the one unmarshaled from the url ?

Reply:
Some for other structs

Reply:
Good point. I will add them.